pkg/domain/infra/abi/parse/parse.go

```diff
@@ -1,93 +1,11 @@
 package parse
 
 import (
-	"strconv"
-	"strings"
-
 	"github.com/containers/podman/v4/libpod"
-	"github.com/containers/podman/v4/libpod/define"
-	units "github.com/docker/go-units"
-	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
 )
 
 // Handle volume options from CLI.
 // Parse "o" option to find UID, GID, Size.
 func VolumeOptions(opts map[string]string) ([]libpod.VolumeCreateOption, error) {
```

Comment:
Why do you keep the old function around?
Can you just update the callers to use the new place?

Reply:
Or fix up this function to be able to be used in new location?

Reply:
Removed this redundant function in an additional commit.